src/main/scala/org/broadinstitute/dsde/firecloud/utils/TSVFormatter.scala

```diff
@@ -68,14 +77,17 @@ object TSVFormatter {
   }
 
   /**
-    * JsValues are double-quoted. Need to remove them before putting them into a cell position
+    * JsString will double-quote strings via JsValue.toString(),
+    * so handle JsString differently than other JsValues.
     *
-    * @param value The JsValue to remove leading and trailing quotes from.
-    * @return Trimmed string value
+    * @param value The JsValue to stringify
+    * @return Resultant string value
     */
   def cleanValue(value: JsValue): String = {
-    val regex = "^\"|\"$".r
-    regex.replaceAllIn(value.toString(), "")
+    value match {
+      case s:JsString => s.value
+      case x => x.toString()
+    }
```

Comment:
this change feels more accurate to the intention of the method, and it is necessary to allow the double-quoted values through to the TSV output.